### ISSUE TITLE
Using Flambo as a checkout dependency

### ISSUE THREAD

**Issue by jogjayr** [yieldbot/flambo#21]
TL;DR I'm working on a project that uses Flambo. As I work, I would like to be able to add new features to Flambo as I need them. They could then eventually be submitted back to Flambo. 

I am trying to use checkout dependencies in Leiningen but have so far been foiled because of the slash (/) in the project's name (checkout dependencies require you to make a symlink to a checkout of the repo)

More info here: http://stackoverflow.com/questions/26707540/leiningen-checkouts-when-library-has-a-slash-in-its-name

It's possible I'm doing it wrong, in which case I would really appreciate it if you could show me the right way.
